test/bzmesh_symmetry.jl

```diff
@@ -15,14 +15,14 @@
         basis = PlaneWaveBasis(model_nosym; Ecut=5, case...)
         DFTK.check_group(basis.symmetries)
 
-        scfres = self_consistent_field(basis; is_converged=DFTK.ScfConvergenceDensity(1e-10))
+        scfres = self_consistent_field(basis; tol=1e-10)
```

Comment:
Don't remove this everywhere, because if we refactor it later, we don't see things breaking.